health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FacilityPopup.js

```
@@ -370,64 +335,136 @@ const FacilityPopUp = ({ details, onClose }) => {
+                  />
+                </Card>
               </div>
-              <div className="card-container" style={{ border: "1px solid #D6D5D4", borderRadius: "3px" }}>
+              <Card className="facility-popup-table-card" type={"primary"}>
                 {selectedRows.length > 0 && (
-                  <div className="card-container" style={{ display: 'flex', justifyContent: 'space-between', alignItems: 'center', width: '100%', borderBottom: '1px solid #D6D5D4', padding: '1rem' }}>
-                    <div>
-                      {selectedRows.length} {selectedRows.length === 1 ? t('MICROPLAN_SELECTED') : t('MICROPLAN_SELECTED_PLURAL')}
+                  <div className="selection-state-wrapper">
+                    <div className="svg-state-wrapper">
+                      <SVG.DoneAll width={"1.5rem"} height={"1.5rem"} fill={"#C84C0E"}></SVG.DoneAll>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)